code/src/utest/pcgen/base/util/WeightedCollectionTest.java

```diff
@@ -47,13 +48,12 @@
 
 	private WeightedCollection<Integer> wc;
 
-	@Before
+	@BeforeEach
 	public void setUp()
 	{
 		wc = new WeightedCollection<>();
 	}
 
-	@SuppressWarnings("unused")
```

Comment:
Why remove this?

Reply:
in test code its mostly useless. Now fixed with `assertThrows`